Server/Content/Sources/App/routes.swift

```diff
@@ -0,0 +1,24 @@
+import Fluent
+import Vapor
+
+func routes(_ app: Application) throws {
+    app.get { req in
+        return "It works!"
+    }
+
+    app.get("hello") { req -> String in
+        return "Hello, world!"
+    }
+
+    let textController = TextController()
+    app.get("texts", use: textController.all)
+    app.get("texts", ":textID", use: textController.getById)
+    app.post("texts", use: textController.create)
+    app.delete("texts", ":textID", use: textController.delete)
```

Comment:
You can declare a route collection instead, to group all of the [related endpoints together.](https://docs.vapor.codes/4.0/routing/#route-groups)
Also I think it's better done inside of controllers, check out how I did it in Gateway service